src/ducks/billsMatching/Linker/Linker.js

```diff
@@ -0,0 +1,377 @@
+const bluebird = require('bluebird')
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.